### ISSUE TITLE
Missing dependency on typescript

### ISSUE THREAD

**Issue by interfect** [codechecks/monorepo#70]
`@codechecks/client@0.1.10` has a dependency on `node-ts` `^8.0.2`. But `node-ts@8.0.2` has a peer dependency on `typescript` `>=2.0`, and `node-ts@8.6.0` (which is what gets installed today) has a peer dependency on `typescript` `>=2.7`.

Since the CodeChecks client module is the one that depends on the module with the peer dependency, as I understand it the CodeChecks client module is the one that is responsible for depending on it as a non-peer dependency. There should be a dependency on `typescript` `>=2.7` to satisfy `node-ts`, and this dependency needs to be bumped if/when `node-ts` releases a new version that matches `^8.0.2` and bumps its peer dependency version.

**Comment by interfect** [comment]
To reproduce:

```
npm install --save @codechecks/client
npm ls
```

**Comment by timocov** [comment]
I don't think that `ts-node` should be in dependencies, it looks strange for me. `ts-node` usually is used to run typescript code without transpiling _locally_. @krzkaczor can you please take a look?
